spec/models/book_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  subject(:book) { described_class.new(gender: gender, author: author, image: image, title: title, editor: editor, year: year) }
```

Comment:
Dont change the line length because of this, make it more easy to read by adding new lines